.github/workflows/linkcheck-changed-files.yaml

```diff
@@ -2,7 +2,7 @@ name: "LinkCheck - Changed files"
 "on":
   push:
     branches:
-      - master
+      - main
     tags:
       - "**"
```

Comment:
I think we can leave the tags as it is cause we are using tags even is just one for now.